src/configure/templateInputHelper/InputControl.ts

```diff
@@ -103,6 +104,21 @@ export class InputControl {
         }
     }
 
+    public getPropertyValue(propertyName: string, inputs?: StringMap<any>): string {
+        var properties = this.inputDescriptor.properties;
+        var value: string;
+
+        if (!!properties) {
+            value = properties[propertyName];
+            if (!!value && !!inputs && typeof value === "string") {
```

Comment:
change the condition to:
`if( !!value && !!inputs ) {
  return MustacheHelper.render(value, { inputs: inputs }); 
}`

and remove else part.